gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -207,6 +207,27 @@ final void setManifestFile(File manifestFile) {
     @TaskAction
     final void createManifest() throws Exception {
         validateProjectVersion();
+        jsonMapper.writeValue(
+                getManifestFile(),
+                SlsManifest.builder()
+                        .manifestVersion("1.0")
+                        .productType(productType.get())
+                        .productGroup(serviceGroup.get())
+                        .productName(serviceName.get())
+                        .productVersion(getProjectVersion())
+                        .putAllExtensions(manifestExtensions.get())
+                        .putExtensions("product-dependencies", computeProductDependencies())
+                        .putExtensions("diagnostics", findSupportedDiagnosticTypesFromClasspath())
+                        .build());
+    }
+
+    // See the internal sls-spec's debug.md document
+    private Diagnostics.SupportedDiagnostics findSupportedDiagnosticTypesFromClasspath() {
+        return Diagnostics.loadFromConfiguration(
+                getProject(), getProject().getConfigurations().getByName("runtimeClasspath"));
+    }
```

Comment:
I think I need to declare that the runtimeClasspath configuration is an `@Input` of this task, otherwise people could change stuff and find their manifest is out of date.